### PR TITLE
prevent LUSCUS from installing it's configuration into $HOME/.luscus

### PR DESCRIPTION
**Pull request by boegel**
I couldn't find another way than using a patch file, the `$HOME/.luscus` is hardwired, no way to override it using an option to `cmake`...

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3107.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/323232c55d2a35056e73ecd0e813e526 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a04664ed615cbf24f8936c4d42d2ac68 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/7c443d6f922a1c5fb8dd53a81cdb4bd6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (2 easyconfigs in this PR)
node2543.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fd14ddbf15429a0e1c1c67b46668fa8f for a full test report.

File: easybuild/easyconfigs/l/LUSCUS/LUSCUS-0.8.6-foss-2018b.eb [hunk 1]
```diff
@@ -29,7 +33,9 @@ dependencies = [
 
 sanity_check_paths = {
     'files': ['bin/luscus'],
-    'dirs': [],
+    'dirs': ['config'],
 }
 
+modextravars = {'LUSCUS_DIR': '%(installdir)s/config'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/LUSCUS/LUSCUS-0.8.6-foss-2018b.eb]: why not `modextrapath`?

Comment (boegel) [reply easybuild/easyconfigs/l/LUSCUS/LUSCUS-0.8.6-foss-2018b.eb]: Because it's a pointer to a single directory, it's not a `$PATH`-like variable

**Pull request merged**